docs/nitpick-exceptions

```diff
@@ -131,6 +131,8 @@ py:class reference target not found: (k, v)
 py:class EllipsisType
 py:class ModuleType
 py:class Real
+# numpy
+py:class np.number
```

Comment:
Is this worth an issue over at numpy?

Reply:
I have not checked if the problem is in NumPy documentation or if it is caused by the known Sphinx bugs linked to a few lines above.